alerts/cluster-monitoring-operator/ThanosRuleEvaluationLatencyHigh.md

```diff
@@ -0,0 +1,46 @@
+# ThanosRuleEvaluationLatencyHigh
+
+## Meaning
+
+The [Thanos Ruler][1] is deployed when user workload monitoring is enabled. It
+allows alerting rules deployed as part of user workload monitoring to query
+both the Prometheus instance responsible for cluster components as well the
+user-workload Prometheus instance. This alert is triggered when the
+Thanos Ruler takes longer time to evaluate the alert.
```

Comment:
I'd keep the description shorter and avoid mentioning that Thanos Ruler queries both platform and UWM prometheus.

```suggestion
The alert fires when Thanos Ruler misses rule evaluations due to slow rule group processing. The alert is only about the user-defined recording and alerting rules.
```